cmd/podman/create.go

```diff
@@ -884,17 +884,16 @@ func joinOrCreateRootlessUserNamespace(createConfig *cc.CreateConfig, runtime *l
 			if err != nil {
 				return false, -1, err
 			}
-			pid, err := ctr.PID()
+
+			data, err := ioutil.ReadFile(ctr.Config().ConmonPidFile)
 			if err != nil {
-				return false, -1, err
+				return false, -1, errors.Wrapf(err, "cannot read conmon PID file %q", ctr.Config().ConmonPidFile)
 			}
-			if pid == 0 {
-				if createConfig.Pod != "" {
-					continue
-				}
-				return false, -1, errors.Errorf("dependency container %s is not running", ctr.ID())
+			conmonPid, err := strconv.Atoi(string(data))
```

Comment:
Did we drop the `pod == 0` check here, or is it handled in `JoinDirectUserAndMountNS()`

Reply:
it should fail as it is not running, but I am going to add it again as it is clearer to understand.

Thanks to catch it